src/includes/performance/distributed-tracing/dotnet.mdx

```diff
@@ -0,0 +1,45 @@
+## Distributed Tracing
```

Comment:
`distributed-tracing` is under a `PlatformSection` section with only `java` as supported, either we move it outside of Java or add .NET as a supported platform, you cannot see this page right now.

https://raw.githubusercontent.com/getsentry/sentry-docs/master/src/platforms/common/performance/custom-instrumentation.mdx

Reply:
Thanks, I added a separate section with `supported={["dotnet"]}` because there's only one page we currently support.